pkg/controller/metrics/metrics.go

```diff
@@ -97,8 +113,35 @@ func (mc *Calculator) Start(stopCh <-chan struct{}) error {
 				installedTotal = installedTotal + 1
 			}
 		}
+		// jobs metrics
+		jobs := &batchv1.JobList{}
+		err = mc.Client.List(context.Background(), &client.ListOptions{}, jobs)
+		if err != nil {
+			log.WithError(err).Error("error listing install jobs")
+		}
+		mcLog.Infof("calculating jobs metrics")
+		installJobsTotal := 0
+		uninstallJobsTotal := 0
+		// legacyJobs is the variable that tells us about the number of old jobs that might exist in production
+		legacyJobs := 0
```

Comment:
Per the above query change you'll no longer be seeing legacy jobs so lets remove this.